src/Dapr.Client/GetWorkflowResponse.cs

```diff
@@ -19,8 +19,12 @@ namespace Dapr.Client
         /// <summary>
         /// Initializes a new <see cref="GetWorkflowResponse" />.
         /// </summary>
-        /// <param name="instanceId">The instance ID assocated with this response.</param>
-        /// <param name="startTime">The time at which the workflow started executing.</param>
-        /// <param name="metadata">The response metadata.</param>
-        public record GetWorkflowResponse(string instanceId, DateTime startTime, IReadOnlyDictionary<string, string> metadata);
+        /// <param name="instanceId">The instance ID associated with this response.</param>
+        /// <param name="workflowName">The name of the workflow associated with this response.</param>
+        /// <param name="createdAt">The time at which the workflow started executing.</param>
+        /// <param name="lastUpdatedAt">The time at which the workflow started executing.</param>
+        /// <param name="runtimeStatus">The current runtime status of the workflow.</param>
+        /// <param name="properties">The response properties.</param>
+        public record GetWorkflowResponse(string instanceId, string workflowName, DateTime createdAt,
```

Comment:
Nit: For method definitions, if you want to split the line, you should put each parameter on a new line. All of which should just have 1 indent.